understanding/20/change-on-request.html

```diff
@@ -10,12 +10,13 @@ <h1>Understanding Change on Request</h1>
       
 		<section id="brief">
 			<h2>In brief</h2>
-			<dl>
-				<dt>Objective</dt><dd>Users have full control of major content changes</dd>
-				<dt>Author task</dt><dd>Provide ways for users to trigger or turn off changes of context</dd>
-			</dl>
+         <dl>
+		         <dt>Goal</dt><dd>Users have full control of major content changes.</dd>
+		         <dt>What to do</dt><dd>Provide ways for users to trigger or turn off changes of context.</dd>
+			      <dt>Why it's important</dt><dd>Content that behaves predictably is especially important to people with disabilities.</dd> 
```

Comment:
Another one where the form could match better. Something like: "People with disabilities can predict how the content/site/page/app/insert-better-noun will behave."